pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -4777,6 +4777,59 @@ var _ = Describe("VirtualMachine", func() {
 
 					testutils.ExpectEvents(recorder, common.FailedCreateVirtualMachineReason)
 				})
+
+				It("should not expand and update VM with referencePolicy reference", func() {
+					testutils.UpdateFakeKubeVirtClusterConfig(kvStore, &v1.KubeVirt{
+						Spec: v1.KubeVirtSpec{
+							Configuration: v1.KubeVirtConfiguration{
+								Instancetype: &v1.InstancetypeConfiguration{
+									ReferencePolicy: pointer.P(v1.Reference),
+								},
+							},
+						},
+					})
+
+					vm.Spec.Instancetype = &v1.InstancetypeMatcher{
+						Name: instancetypeObj.Name,
+						Kind: instancetypeapi.SingularResourceName,
+					}
+					vm, err := virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Create(context.TODO(), vm, metav1.CreateOptions{})
```

Comment:
```suggestion
					vm, err := virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Create(context.Background(), vm, metav1.CreateOptions{})
```

nit: This is still under discussion in general